config.tfvars

```diff
@@ -37,4 +37,15 @@ db_instance_class    = "db.t3.micro"
 db_allocated_storage = 1000
 db_iops              = 1000
 
-
+# Bamboo license
+# To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` and keep the below line commented out
```

Comment:
```suggestion
# To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_` (i.e. `TF_VAR_bamboo_license`) and keep the below line commented out
```